tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist = py27,py36,static,pidiff
+
+[testenv]
+deps=-rtest-requirements.txt
+commands=pytest -v {posargs}
+whitelist_externals=sh
+
+[testenv:py27]
+deps=
+	# Note: we need to explicitly list requirements.txt here
+	# so it's processed at the same time as the constraints file
+	-rrequirements.txt
+  -rtest-requirements.txt
```

Comment:
Indentation on line 14 won't be an issue?

Reply:
+1

Reply:
ah, yeah, let me fix that. Sometimes for some reason vim decide to turn off expandtabs option